### ISSUE TITLE
Release architectures

### ISSUE THREAD

**Issue by jmthackett** [ssbc/patchwork#471]
Hello!

First of all, thanks to everyone who has made this project. It looks fun and useful and ticks all my boxes for "more of this, please!"

I'm not currently able to use the software though, as I do my daily computing split between a 32bit Windows PC and a Raspberry Pi 3. I don't have entirely mad reasons for this setup: it is a long term experiment in computing on the smallest possible power budget, but it does prohibit me from trying out what looks like great software from time to time. If a build for either platform would appear, that'd be really helpful to me. I suspect a build for the Pi would be good as those kinds of users are likelier than the 32bit Windows crowd.

TL;DR Asking for a build that'll run on a Raspberry Pi 3, or 32bit Windows.

**Comment by ahdinosaur** [comment]
hey @jmthackett, thanks! i appreciate your experiment and i know there are others in the network who can relate. :smile:  

you might be interested in @clehner's [`patchfoo`](https://git.scuttlebot.io/%25YAg1hicat%2B2GELjE2QJzDwlAWcx0ML%2B1sXEdsWwvdt8%3D.sha256), an alternative to Patchwork "designed for use on low-power and low-resource computers".
